utils/cliutils/commandsflags.go

```diff
@@ -544,6 +545,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  retries,
 		Usage: "[Default: " + strconv.Itoa(Retries) + "] Number of HTTP retries.` `",
 	},
+	retryWaitTime: cli.StringFlag{
+		Name:  retryWaitTime,
+		Usage: "[Default: 0]  Number of seconds/milli-seconds to sleep between retries.` `",
```

Comment:
Redundant white-space before * Number*.

Number of seconds/milli-seconds to sleep between retries.
-->
Number of seconds or milliseconds to wait between retries. The numeric value should either end with s for seconds or ms for milliseconds.